### PR TITLE
Fuzz: cleanup the Makefiles

### PR DESCRIPTION
**Pull request by miekg**
Move from things from the main Makefile to Makefile.fuzz and make things
smaller and select fuzz targets automatically, just plop a fuzz.go in
the plugin's directory.

### CODE REVIEW

**Comment by miekg** [comment]
fixing things later today

File: Makefile [hunk 1]
```diff
@@ -46,23 +46,10 @@ ifeq ($(TEST_TYPE),coverage)
...
-        chmod a+x fuzzit; \
-        ./fuzzit create job --type $(FUZZIT_TYPE) coredns/cache ./cache; \
-        ./fuzzit create job --type $(FUZZIT_TYPE) coredns/chaos ./chaos; \
-        ./fuzzit create job --type $(FUZZIT_TYPE) coredns/file ./file; \
-        ./fuzzit create job --type $(FUZZIT_TYPE) coredns/rewrite ./rewrite; \
-        ./fuzzit create job --type $(FUZZIT_TYPE) coredns/whoami ./whoami; \
-        ./fuzzit create job --type $(FUZZIT_TYPE) coredns/corefile ./corefile; \
-    fi;
+	if [ "$(TRAVIS_PULL_REQUEST)" = "false" ] || [ "$(FUZZIT_TYPE)" = "local-regression" ]; then \
+		$(MAKE) -f Makefile.fuzz install; LIBFUZZER=YES; \
```
Comment (yevgenypats) [inline Makefile:50]: I think the make -f Makefile.fuzz $TARGETS is now missing this why the [build](https://travis-ci.org/coredns/coredns/jobs/573361289#L812) fails as the targets are not built

Comment (miekg) [reply Makefile:50]: yes, this is proofing more annoying than anticipated :(

**Comment by miekg** [comment]
I fucked up this PR, closing and retrying in smaller steps.

**Pull request closed without merging**